### DISCUSSION TITLE
テーブルを横スクロール可能にできないか

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#444] [要望・アイデア・不具合]
横に長いテーブルをスマホなどで見た場合、狭いセルにテキストが折り返され、とても見づらいことになります。

![image](https://user-images.githubusercontent.com/20086673/223142233-bb14094b-2e4c-4dd0-bf78-8eb80de257cd.png)

テーブルを横スクロール可能にできれば解消できそうですが、どうでしょうか？

**Comment by tanaka-asami415** [comment]
@munierujp 
ご意見ありがとうございます！🙏こちら検討します！

**Comment by tanaka-asami415** [comment]
@munierujp 
長らくお待たせして申し訳ございません。

テーブルのスクロールについて調査および検討したところ、現時点での実装は難しいという結論に至りました。
理由としては、Qiitaでは多種多様なテーブルの使い方がされており、
すべてのテーブルに対して最適なスクロールとセルの折返しなどを実装するのは難しいと判断しました。
お待たせしたにもかかわらず、今回はご希望に添えず申し訳ありません🙇

引き続きサービス改善に努めますので、ご意見ご要望をお寄せいただけると幸いです🙏
